admin/base/views.py

```diff
@@ -8,7 +8,9 @@
 
 @login_required
 def home(request):
-    context = {'user': request.user}
+    context = {
+        'user': request.user,
+    }
     return render(request, 'home.html', context)
```

Comment:
Because `django.template.context_processors.request` is included in your context processors request is always in your context so you can remove 11-13 and do
```python
return render(request, 'home.html')
```
If that doesn't work send context as an empty object.

`{{ request.user }}` should work in the template.